### DISCUSSION TITLE
Platform specific XAML Page

### DISCUSSION THREAD

**Discussion by jhertel0** [dotnet/maui#6012] [Q&A]
We are trying out MAUI, we are wanting to use the WinUI community toolkit for our windows specific pages, which for now will probably be most of the app, but in the future we will convert them to mobile friendly as needed. Is there any way to have XAML pages only for a certain platform, such as windows? 

I've tried adding them into the Windows platform folder but I can't get it to work.

**Comment by CDboyOne** [comment]
Same issue. I want to know if it is solved.

**Reply by FM1973** [reply]
I solved this by making a contentview (code only) and using templatecontrols per platform.
